app/main.py

```diff
@@ -2,0 +1,43 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100
+                 ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    @classmethod
+    def remove_dead(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    @classmethod
+    def __str__(cls) -> str:
+        return str([repr(animal) for animal in cls.alive])
+
+    @classmethod
+    def print_alive(cls) -> list:
+        return cls.alive
```

Comment:
The method `print_alive` returns a list of alive animals, but the name suggests it prints something. Consider renaming it to `get_alive` to better reflect its functionality.